content/en/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Migrating from dockershim to other container runtimes"
+weight: 10
+---
+
+<!-- overview -->
+
+This section presents information you need to know when migrating from
+dockershim to other container runtimes.
+
+Since the announcement of [dockershim deprecation] in Kubernetes 1.20, there
+were questions on how this will affect various workloads and Kubernetes
+installations. You can find these links useful to understand the problem better:
+
+- [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/)
+- [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+
+We highly encourage you to consider various migration options. Check out
+[container runtimes] section to know your options. Make sure to [report
+issues](https://github.com/kubernetes/kubernetes/issues) you encountered with
+the migration. So the issue can be fixed timely and your cluster would be
+ready for dockershim removal.
+
+[dockershim deprecation]: /blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation
+[container runtimes]: /docs/setup/production-environment/container-runtimes/
+
+
+## {{% heading "whatsnext" %}}
+
+{{< note >}}
+
+The following sections may be added in future:
+
+- Dockershim deprecation and Windows
+- Known issues of non-dockershim runtimes
+- Transition from docker CLI to crictl
+- Building images on kubernetes without Docker
+
+{{< /note >}}
```

Comment:
We usually [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) in published documentation. It's OK to say “foo _has been_ deprecated” but we prefer not to hint “we might document this more some day”.

Reply:
I will remove this before merging this PR